tests/unit/test_models.py

```diff
@@ -83,11 +81,21 @@ def __init__(self, *args, **kwargs):
 
     loaded_blob = engine._load(Blob, blob)
 
-    assert init_called
-
+    assert init_called is False
     assert loaded_blob.id == "foo"
     assert loaded_blob.data == "data"
-    assert not hasattr(loaded_blob, "extra_field")
+
+    same_obj = Blob()
+    same_obj.id = "foo"
+    same_obj.data = "data"
+    same_obj.extra_field = "0.125"
+    engine.save(same_obj)
```

Comment:
What does this code (and the code below it) exercise?  Since it's a unit test the `save` is connected to a mock operation and won't actually be returned back with `engine.load`.  Additionally, the `engine._load` call won't interact with the existing instance `loaded_blob` or the value saved (since it's still unpacking the `blob` dict above).

I think you can simplify the changes to this method to check `assert not init_called` and leave the rest, although I may be missing something.

Reply:
Yep, you are correct.  Will fix.